.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -10,23 +10,46 @@ assignees: ''
 
 The release number or commit hash of the version you're using.
 
-## Operating system & version
+## \[Optional\] Relevant system infromation
 
-Ex: Windows 10, Ubuntu 18.04, iOS 14.
+If you cannot get Bevy to build or run on your machine, please include:
 
-## What you did
+- the Rust version you're using (you can get this by running `cargo --version`)
+  - Bevy relies on the "latest stable release" of Rust
+  - nightly should generally work, but there are sometimes regressions: please let us know!
+- the operating system or browser used, including its version
+  - e.g. Windows 10, Ubuntu 18.04, iOS 14
+
+If your bug is rendering-related, copy the adapter info that appears when you run Bevy.
 
-The steps you took to uncover this bug. Please list full reproduction steps if
-feasible.
+```ignore
+`AdapterInfo { name: "NVIDIA GeForce RTX 2070", vendor: 4318, device: 7938, device_type: DiscreteGpu, backend: Vulkan }`
+```
 
-## What you expected to happen
+You should also consider testing the examples of our upstream dependencies to help isolate any setup-specific issue:
 
-What you think should've happened if everything was working properly.
+- [`wgpu`](https://github.com/gfx-rs/wgpu) for rendering problems
+- [`winit`](https://github.com/rust-windowing/winit) for input and window management
+- [`gilrs`](https://docs.rs/gilrs/latest/gilrs/) for gamepad inputs
+
+## What you did
 
-## What actually happened
+Please provide a runnable code snippet or link if you can.
```

Comment:
This current wording could make it seem like providing a code example is the suggested or even the only way you could go about this. This is a place where a little more verbosity might be warranted.

```suggestion
Describe how you arrived at the problem. If you can, consider providing a code snippet or link.
```

This wording feels a lot more friendly. It gives you options and is not upfront about asking for a code example.